src/commands/utils/webviews.ts

```diff
@@ -0,0 +1,119 @@
+import * as fs from 'fs';
+import * as htmlhandlers from "handlebars";
+import { HelperDelegate } from 'handlebars';
+import * as path from 'path';
+import * as vscode from 'vscode';
+
+export enum HtmlHelper {
```

Comment:
If someone points out this is unnecessary, I'll agree. I was trying to match the existing behaviour in which different commands would register different sets of helpers. But since nothing is ever unregistered, with enough use all the helpers will get registered in the end anyway. It'd probably be clearer if we just registered them all at the start, and take the responsibility away from the commands entirely.

But I'd prefer to get input from others before going ahead and doing that.

Reply:
Discussed with @Tatsinnit and removed.